src/DssSpell.t.sol

```diff
@@ -398,8 +398,38 @@ contract DssSpellTest is DssSpellTestBase {
         new DssSpell();
         uint256 endGas = gasleft();
         uint256 totalGas = startGas - endGas;
+
+        // Warn if deploy exceeds block target size
+        if (totalGas >= 15 * MILLION) {
+            emit log("Warn: deploy gas < average block target");
+            emit log_named_uint("    deploy gas", totalGas);
+            emit log_named_uint("  block target", 15 * MILLION);
+        }
+
         // Fail if deploy is too expensive
-        assertLe(totalGas, 15 * MILLION);
+        assertLe(totalGas, 30 * MILLION, "testDeployCost/DssSpell-exceeds-max-block-size");
+    }
+
+
+    // Fail when contract code size exceeds 24576 bytes (a limit introduced in Spurious Dragon).
+    //  This contract may not be deployable on mainnet.
```

Comment:
```suggestion
    // This contract may not be deployable.
```
Same for Goerli right?

Reply:
Yeah, I think goerli has the limit now. It didn't before.